testpar/t_filters_parallel.c

```diff
@@ -774,15 +808,17 @@ test_write_filtered_dataset_single_no_selection(void)
     data_size        = sel_dims[0] * sel_dims[1] * sizeof(*data);
     correct_buf_size = dataset_dims[0] * dataset_dims[1] * sizeof(*correct_buf);
 
-    data = (C_DATATYPE *)HDcalloc(1, data_size);
-    VRFY((NULL != data), "HDcalloc succeeded");
+    if (mpi_rank != WRITE_SINGLE_NO_SELECTION_FILTERED_CHUNKS_NO_SELECT_PROC) {
+        data = (C_DATATYPE *)HDcalloc(1, data_size);
```

Comment:
Not a huge deal, but there's no need to cast the results of allocations.

Reply:
Very true. Just copy and pasted from old code that was written when I was less familiar with C conventions.